{"issue_url": "https://api.github.com/repos/rails/rails/issues/9302", "repo": "rails", "body": "@spastorino That seems to be in a different namespace. That raises `ActiveSupport::MessageEncryptor::InvalidMessage`, which is already defined. \r\n\r\n`InvalidMessage` error in the `MessageVerifier` namespace doesn't appear to be used anywhere.", "_id": {"$oid": "5237ead1bd3543c151009a74"}, "issue_id": 9302, "url": "https://api.github.com/repos/rails/rails/issues/comments/13712134", "html_url": "https://github.com/rails/rails/pull/9302#issuecomment-13712134", "updated_at": "2013-02-18T09:03:52Z", "user": {"subscriptions_url": "https://api.github.com/users/felixclack/subscriptions", "events_url": "https://api.github.com/users/felixclack/events{/privacy}", "repos_url": "https://api.github.com/users/felixclack/repos", "gists_url": "https://api.github.com/users/felixclack/gists{/gist_id}", "url": "https://api.github.com/users/felixclack", "login": "felixclack", "gravatar_id": "1abe821790c5a8516b6f5baafff5fa16", "html_url": "https://github.com/felixclack", "following_url": "https://api.github.com/users/felixclack/following{/other_user}", "received_events_url": "https://api.github.com/users/felixclack/received_events", "organizations_url": "https://api.github.com/users/felixclack/orgs", "avatar_url": "https://2.gravatar.com/avatar/1abe821790c5a8516b6f5baafff5fa16?d=https%3A%2F%2Fidenticons.github.com%2F9f8684e630c4c30cad7b1f0935cd62ab.png", "starred_url": "https://api.github.com/users/felixclack/starred{/owner}{/repo}", "id": 3126, "followers_url": "https://api.github.com/users/felixclack/followers", "type": "User"}, "created_at": "2013-02-18T09:03:52Z", "id": 13712134, "owner": "rails"}
